.github/workflows/CI.yaml

```
@@ -40,4 +41,4 @@ jobs:
         python -m pip install . --progress-bar off
```

```suggestion
        uv pip install . --no-build-isolation --no-deps -v
```
Install with the dependencies explicitly provided by the synced environment.